src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -230,10 +225,37 @@ public static JsonRpc Attach(Stream sendingStream, Stream receivingStream, objec
         }
 
         /// <summary>
+        /// Adds the specified targets as possible objects to invoke when incoming messages are received.
```

Comment:
Either here or in the `<remarks>` section, can you explain the relationship these targets have with each other, and how order is significant?